convert2rhel/systeminfo.py

```diff
@@ -33,11 +35,26 @@
 from convert2rhel.toolopts import tool_opts
 
 
+# Supported conversion path to RHEL
+RELEASE_VER_MAPPING = {
```

Comment:
Should we no longer require to have the recent version, but any version available in this dict?
Should we update the docs to refer to this dict instead of duplicating this info?

Reply:
As in to say, any releases outside the mapping will not be converted by default. Should there be a new 8.5 RHEL and CentOS release and we haven't updated this mapping, the conversion will also fail (by default)

Not saying that will happen if we update it, but just extending a) a bit

Reply:
Agreed. For that reason I suggested to update the mapping to include 8.5 (https://github.com/oamg/convert2rhel/pull/299#discussion_r648204572).